c/ldv-memsafety-bitfields/test-bitfields-2.1_false-valid-free.c

```diff
@@ -7,8 +7,10 @@ struct A {
 	unsigned char a;
 	unsigned char b:2;
 	unsigned char c:2;
+	unsigned char pad1[2];
```

Comment:
The alignment was first  `[a*8,_*24,b*2,c*2,_*20,d*32,e*8,_*24]` (in bits, not packed, full length=4Byte), 
then `[a*8,b*2,c*2,d*32,e*8,_*12]` (in bits, packed, full length=2Byte) 
and is now `[a*8,b*2,c*2,pad1*16,d*32,e*8,pad2*24,_*4]` (in bits, packed, full length=3Byte).
Is this correct?
Because memcpy copies 4Byte, it would read/write to invalid memory now.
In that case the previous solution (before the change) would have been correct, wouldn't it?